src/osd/OSD.h

```diff
@@ -15,6 +15,11 @@
 #ifndef CEPH_OSD_H
 #define CEPH_OSD_H
 
+#if !defined(BOOST_MPL_LIMIT_LIST_SIZE)
+#define BOOST_MPL_CFG_NO_PREPROCESSED_HEADERS
```

Comment:
Maybe this isn't needed.

Reply:
I removed the use of this define and the code wouldn't compile.

Reply:
@tchaikov do you know a way to avoid these repeated definitions?

Reply:
Created common PeeringState function